examples/mapviewer.py

```diff
@@ -17,28 +17,33 @@
 along with this program; if not, see <http://www.gnu.org/licenses/>.
 """
 
-import sys
-import os.path
 import random
-from math import pi
-from gi.repository import Gtk
-from gi.repository import Gdk
-from gi.repository import GdkPixbuf
-from gi.repository import GObject
+import gi
 
-GObject.threads_init()
-Gdk.threads_init()
+gi.require_version('Gtk', '3.0')
+gi.require_version('OsmGpsMap', '1.0')
+from gi.repository import Gtk  # noqa
+from gi.repository import OsmGpsMap as osmgpsmap  # noqa
 
-from gi.repository import OsmGpsMap as osmgpsmap
-print "using library: %s (version %s)" % (osmgpsmap.__file__, osmgpsmap._version)
+from gi.repository import Gdk  # noqa
+from gi.repository import GdkPixbuf  # noqa
+from gi.repository import GObject  # noqa
+from gi.repository import GLib  # noqa
```

Comment:
Is it important that you make all separate imports here? Would a combined import like that be a (style) issue?

```python
import gi

gi.require_version('Gdk', '3.0')
gi.require_version('Gtk', '3.0')
gi.require_version('OsmGpsMap', '1.0')

from gi.repository import (
    Gdk, GdkPixbuf, GLib, GObject, Gtk, OsmGpsMap as osmgpsmap
)  # noqa
```

Reply:
Done.